backend-tests/tests/test_auditlogs.py

```diff
@@ -241,7 +251,7 @@ def get_auditlog_time(oid: str):
                 evt = evt_user_create(user, uid, uuidv4 + "@acme.com")
                 oid = uid
             else:
-                d, a = make_deployment(user.token)
+                d, a = make_deployment(user.token, device.id)
```

Comment:
Nitpick: I'd use opportunity and consider renaming `d` and `a` variables to something more understandable ;-)